### DISCUSSION TITLE
新北無人機抓違章　宜蘭智慧消防引導打火動線｜智慧城市懶人包#51

### DISCUSSION THREAD

**Discussion by lukiea** [1101-GS3073-C/Python-Community-GS3073-C#714] [General]
https://futurecity.cw.com.tw/article/962
消防員辛苦了，智慧導航不讓你們在火場中走錯路

**Comment by yjpxi** [comment]
智慧消防系統能夠在火災發生時迅速提供最有效的打火動線，協助消防員快速到達火災現場，進行更有效的滅火和救援操作，從而提高救援效率，減少損失，但是如果過度智慧消防系統，系統出現故障或數據錯誤的時候，會影響到救援工作的效果。
